homeassistant/components/light/rfxtrx.py

```diff
@@ -8,10 +8,16 @@
 """
 import logging
 import homeassistant.components.rfxtrx as rfxtrx
+import RFXtrx as rfxtrxmod
```

Comment:
These have to stay in `setup_platform`. If you need this data somewhere else, import it again there.